src/data/www/php71/dbox-tests/src/App/Instances/Model/InstanceModel.php

```diff
@@ -140,14 +160,14 @@ public function add(array $src)
 	{
 		$filter = new InputFilter;
 
-		$instanceID = $src['php_version'] . $src['user_id'] . $src['pr_id'];
+		$instanceID = $src['php_version'] . 'u' . $src['user_id'] . 'p' . $src['pr_id'];
```

Comment:
Why don't you just separate the values with dashes?

Reply:
I thought this way the instance ID would be more descriptive, after the u user ID and after p PR ID but I can change to dashes as well it works with letters or dashes. Should I change to dashes then?

Reply:
The question is: Does anybody benefit from that extra information? Dashes make a clearer separation optically, which IMO increasies readability.

Reply:
Yeah you're right. I'll make that change then no prob ;)